multigeiger/display.cpp

```diff
@@ -106,28 +106,33 @@ void DisplayGMC(int TimeSec, int RadNSvph, int CPS, bool use_display, bool conne
   if (!isLoraBoard) {
     char output[80];
     int TimeMin = TimeSec / 60;         // calculate number of minutes
-    if (TimeMin >= 999) TimeMin = 999;  // limit minutes to max. 999
+    TimeMin = TimeMin % 999;  // limit minutes to max. 999, roll over
```

Comment:
ah, good idea! 

but: off by one. you need to do % 1000 to get 999 max.

Reply:
uppsi, right you are! I'll update the pull request with all changes soon...